### DISCUSSION TITLE
Subscribe copy of existing feed configurations

### DISCUSSION THREAD

**Discussion by hkcomori** [FreshRSS/FreshRSS#6310] [Ideas]
Is there an easier way to copy configurations of existing feeds?

When I set up scraping, if I could make minor modifications based on a copy of a similar feed, it would make the registration process faster.
Currently, I open two FreshRSS feed configuration windows, side by side, with the existing feeds on one side and the new feeds on the other, and copy them manually.

**Comment by Alkarex** [comment]
At the moment, there is a partial solution to this problem, but the UI should be improved.
It is possible to export any feed individually with OPML, and also to import the OPML again.
When you are on a feed, just add `&a=opml` like `https://freshrss.example.net/i/?get=f_175&a=opml`

We should at least add a link from the feed configuration page. PR welcome

**Reply by math-GH** [reply]
Thanks for the reminder ;)
PR #6312

**Comment by hkcomori** [comment]
Thank you, it works fine.
I had assumed that OPML does not contains scraping configurations.
